lib/ads/client_side_ad.js

```diff
@@ -176,7 +180,14 @@ shaka.ads.ClientSideAd = class {
    * @export
    */
   resize(width, height) {
-    const viewMode = document.fullscreenElement ?
+    let isInFullscreen = false;
+    const video = /** @type {HTMLVideoElement} */(this.video_);
+    if (document.fullscreenEnabled) {
+      isInFullscreen = document.fullscreenElement;
```

Comment:
Please make this `!!document.fullscreenElement` so that isInFullscreen is always boolean.

Reply:
Done!